clouddriver-titus/src/test/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/actions/PrepareTitusDeployActionSpec.groovy

```diff
@@ -126,24 +168,34 @@ class PrepareTitusDeployActionSpec extends Specification {
 
   def "security groups are resolved"() {
     given:
-    TitusDeployDescription description = createTitusDeployDescription()
-    description.securityGroups = ["sg-1", "fancyname"]
+    String sg1Id = mimicker.aws().securityGroupId
+    String sg2Name = mimicker.text().word()
+    String sg2Id = mimicker.aws().securityGroupId
+
+    and:
+    TitusDeployDescription description = createTitusDeployDescription(sg1Id)
+    description.securityGroups = [sg1Id, sg2Name]
 
     when:
     subject.resolveSecurityGroups(saga, description)
 
     then:
-    awsLookupUtil.securityGroupIdExists(_, _, "sg-1") >> true
-    awsLookupUtil.securityGroupIdExists(_, _, "fancyname") >> false
-    awsLookupUtil.convertSecurityGroupNameToId(_, _, "fancyname") >> "sg-2"
+    awsLookupUtil.securityGroupIdExists(_, _, sg1Id) >> true
+    awsLookupUtil.securityGroupIdExists(_, _, sg2Name) >> false
+    awsLookupUtil.convertSecurityGroupNameToId(_, _, sg2Name) >> sg2Id
 
-    description.securityGroups == ["sg-2", "sg-1"]
+    description.securityGroups == [sg2Id, sg1Id]
   }
 
   @Unroll
   def "security groups include app security group (label=#labelValue, desc=#descriptionValue, includesAppGroup=#includesAppGroup)"() {
     given:
-    TitusDeployDescription description = createTitusDeployDescription()
+    String sg1Id = mimicker.aws().securityGroupId
```

Comment:
as a mild bit of test coding style I'd generally prefer to stick these kinds of test data in the `where:` section. I suspect that @robfletcher would have the most valid opinion as senior VP of spock testing though

Reply:
Oh, sure yeah that makes sense... I'll change.